### PR TITLE
Adjusting ports for prod

### PR DESCRIPTION
**Pull request by RMeissnerCC**
Homogenizing nginx configuration

### CODE REVIEW

File: docs/main.rst [hunk 1]
```diff
@@ -137,3 +137,24 @@ Install pre-commit hooks::
...
+    mkdir dbt
+    mkdir superset
+
+and run init again.
+
+Use domain corresponding to the IP of the VM, e.g.,::
+
+    cXYZ-UVW.cloud.gwdg.de
+
+Add `.env` file to VM according to `.env.example`. Take are to add the corresponding domain in the `.env`.
```
Comment (MRuecklCC) [inline docs/main.rst:160]: Grammar?

File: nginx/nginx.conf.dev.template [hunk 2]
```diff
@@ -10,7 +10,40 @@ server {
     listen [::]:80;
     listen 80;
     server_name ${NGINX_HOST};
-    keepalive_timeout 65;
+
```
Comment (MRuecklCC) [inline nginx/nginx.conf.dev.template:13]: Now this file is almost equivalent to `nginx-ssl.conf.template`. The only difference is this block:

```
    location /docs {
        auth_basic "OpenEduHub";
        auth_basic_user_file /etc/nginx/.htpasswd;
        try_files $uri $uri/ =404;
        index index.html;
    }
```
Can we simply drop the dev template and use `nginx-ssl.conf.template` for dev and prod?

File: docs/main.rst (hunk 1 above)
Comment (RMeissnerCC) [reply docs/main.rst:160]: Is now present

File: nginx/nginx.conf.dev.template (hunk 2 above)
Comment (RMeissnerCC) [reply nginx/nginx.conf.dev.template:13]: Good point, thanks for pointing out. I removed the /docs part, because it is unused. Test in dev was succesful

**Review by MRuecklCC — APPROVED** [review]

**Pull request merged**